desktop/core/src/desktop/js/reactComponents/TaskBrowser/TaskBrowser.tsx

```diff
@@ -0,0 +1,434 @@
+        newStatusFilter.running = false;
+      } else {
+        // If any specific status is toggled, set 'all' to false
+        newStatusFilter.all = false;
+      }
+  
+      // If no individual statuses are selected, default to 'all'
+      if (!newStatusFilter.success && !newStatusFilter.failure && !newStatusFilter.running) {
+        newStatusFilter.all = true;
+      }
+  
+      return newStatusFilter;
+    });
+  };
+  
+  
+
+  const filteredTasks = tasks.filter(task => {
+    const taskNameMatch = task.result?.task_name?.toLowerCase().includes(searchTerm);
+    const userIdMatch = task.result?.username?.toString().toLowerCase().includes(searchTerm);
+    const taskIdMatch = task.task_id?.toString().toLowerCase().includes(searchTerm);
+    // const statusMatch = statusFilter === 'all' || task.status.toLowerCase() === statusFilter;
+    const statusMatch = statusFilter.all || 
+                      (statusFilter.success && task.status.toLowerCase() === 'success') ||
+                      (statusFilter.failure && task.status.toLowerCase() === 'failure') ||
+                      (statusFilter.running && task.status.toLowerCase() === 'running');
+
+
+
+    return (taskNameMatch || userIdMatch || taskIdMatch) && statusMatch;
+  });
+
+  const handleKillTask = (taskId) => {
+    axios.post(`/desktop/api2/kill_task/${taskId}/`)
+      .then(response => {
+        const { status, message } = response.data;
+        if (status === 'success') {
+          alert(`Task: ${taskId} has been killed.`);
+        } else if (status === 'info') {
+          alert(`Task: ${taskId} has already been completed or revoked.`);
+        } else {
+          alert(`Task: ${taskId} could not be killed. ${message}`);
+        }
+      })
+      .catch(error => {
+        console.error('Error killing task:', error);
+        alert(`Error killing task: ${error.message}`);
+      });
+  };
+
+  const handleKillSelectedTasks = () => {
+    selectedTasks.forEach(taskId => {
+      handleKillTask(taskId);
+    });
+    setSelectedTasks([]); // Clear the selection after killing the tasks
+  };
+
+  const handleCheckboxChange = (taskId) => {
+    setSelectedTasks(prevSelectedTasks =>
+      prevSelectedTasks.includes(taskId)
+        ? prevSelectedTasks.filter(id => id !== taskId)
+        : [...prevSelectedTasks, taskId]
+    );
+  };
+
+
+
+
+  // Render the table with the tasks data
+  return (
+    <div className="content-panel-inner">
+    <div className="flex-container-style">
+      <button onClick={handleSchedulePopup} className="btn schedule-task-button">Schedule Task</button>
+      <button onClick={handleKillSelectedTasks} className="btn kill-task-button">Kill Task</button>
+      <input
+        type="text"
+        placeholder="Search by task name, user ID, or task ID..."
+        onChange={handleSearchChange}
+        className="button-input-style"
+        style={{ flexGrow: 1 }} // flexGrow allows the input to take up available space
+      />
```

Comment:
This and all inputs below should be ant input components

Reply:
Changed them to ant input components.